adafruit_pyportal.py

```diff
@@ -649,6 +654,33 @@ def image_converter_url(image_url, width, height, color_depth=16):
                                           width, height,
                                           color_depth, image_url)
 
+    def io_push(self, feed, data):
+        # pylint: disable=line-too-long
+        """Push data to an adafruit.io feed
+
+        :param str feed: Name of feed to push data to.
+        :param data: data to send to feed
+
+        """
+        # pylint: enable=line-too-long
+
+        try:
+            aio_username = secrets['aio_username']
+            aio_key = secrets['aio_key']
+        except KeyError:
+            raise KeyError("\n\n")
+
+        wifi = adafruit_esp32spi_wifimanager.ESPSPI_WiFiManager(self._esp, secrets, None)
+        io_connect = RESTClient(aio_username, aio_key, wifi)
+
+        try:
```

Comment:
I do some error catching just below this, please take a look now that I've merged the correct branch and see if it will suffice. I have been running this on my pyportal for a couple weeks now without any  uncaught exceptions, so something must be doing alright :-)